packages/typespec-ts/src/utils/interfaces.ts

```diff
@@ -1,12 +1,13 @@
 import { RLCOptions, SchemaContext } from "@azure-tools/rlc-common";
 import { SdkContext as TCGCSdkContext } from "@azure-tools/typespec-client-generator-core";
-import { ModelProperty } from "@typespec/compiler";
+import { Model, ModelProperty } from "@typespec/compiler";
 import { KnownMediaType } from "./mediaTypes.js";
 
 export interface SdkContext extends TCGCSdkContext {
   rlcOptions?: RLCOptions;
   generationPathDetail?: GenerationDirDetail;
   hasApiVersionInClient?: boolean;
+  nameModelMap?: Map<string, Model[]>;
```

Comment:
you know we have a ContextManager, i was wondering if saving this information into there would be more proper?